### PR TITLE
[Themes] New example for custom themes

### PR DESCRIPTION
**Pull request by romoore**
Supports Primary and Secondary colors using MDCBasicColorScheme.

simulator screen shot - iphone 4s - 2017-08-25 at 17 35 47

### CODE REVIEW

**Review by ajsecord — CHANGES_REQUESTED** [review]
Let's use MDCTextField for this.

**Review by willlarche — CHANGES_REQUESTED** [review]
@romoore , The issue with the text fields is the space they require. This version of text fields requires 63.5 points for non-floating and 84.5 points for floating heights. 

The simplest solution is to increase the square's width and height to 64 and keep it non-floating.

Though I would implement a tap gesture recognizer and / or textFieldShouldReturn to resign first responder. It's a little maddening at the moment if you want to get the keyboard off the screen.

**Comment by romoore** [comment]
Awesome, @willlarche,  thanks for investigating!

**Comment by romoore** [comment]
Added minimum height constraint for both text fields (64pt, keeping non-floating). Added gesture recognizer to resign first responder on both text fields.

File: components/Themes/examples/resources/ThemerCustomSchemePicker.storyboard [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+                            <view contentMode="scaleToFill" horizontalCompressionResistancePriority="1000" verticalCompressionResistancePriority="1000" translatesAutoresizingMaskIntoConstraints="NO" id="wRO-Oc-Kdf" userLabel="primaryView">
+                                <rect key="frame" x="368" y="40" width="288" height="112"/>
+                                <subviews>
+                                    <textField opaque="NO" contentMode="scaleToFill" contentHorizontalAlignment="left" contentVerticalAlignment="center" borderStyle="roundedRect" placeholder="Primary Color" textAlignment="natural" minimumFontSize="17" translatesAutoresizingMaskIntoConstraints="NO" id="S6a-tS-3C0" userLabel="primaryInput" customClass="MDCTextField">
+                                        <rect key="frame" x="0.0" y="24" width="160" height="64"/>
+                                        <constraints>
+                                            <constraint firstAttribute="width" priority="750" constant="160" id="C1a-8M-aWi"/>
+                                            <constraint firstAttribute="height" relation="greaterThanOrEqual" constant="64" id="VQR-Rf-sMK"/>
```
Comment (willlarche) [inline components/Themes/examples/resources/ThemerCustomSchemePicker.storyboard]: Much better. Best would be no height constraint at all.

**Review by willlarche — APPROVED** [review]

**Review by willlarche — APPROVED** [review]

**Pull request merged**